fixtures/anonymous_class_declaration.cpp.sem.uast

```diff
@@ -82,13 +80,10 @@
                            col: 19,
                         },
                      },
-                     IsAlternate: false,
-                     IsOverloadedOperator: false,
+                     Name: "",
```

Comment:
Hmm, that's weird. I think we need to wipe this kind of nodes from the Semantic tree. Should be an empty array in this case.

Reply:
About the `falses` being removed, I changed the driver in this version to only show the "Is..." properties if true because they're a lot and make the AST too big. About the ImplicitNames with the Name to empty, they're mostly duplicated of other properties and actually there is a switch for CDT to not visit those "ImplicitNames", I'll check the fixtures to see if they give some information that other nodes doesn't and if that's not the case remove them.

Reply:
One thing to look out for is that if you don't specify the attributes, the reader may not be able to tell the difference between "this node has `Property:true`" and "`Property` is not relevant to this node". 

In Go, we get away with zero-values because the struct provides a schema. But in a bag-of-props model, it's not intrinsically obvious which attributes _should_ be there (but are defaulted to empty/false/zero). Whether we can get away with it in UAST probably depends on whether the reader knows which attributes go with each node.

Reply:
A lot are pretty language specific (IsVolatile, IsFriend...), but, apart from this drawback, I discovered another one in the previous beta: it complicates annotations mightily by forcing the use of `FieldRoles{Optional...}}` on almost every node. 

So I'm not sure if the reduced AST size is worthy considering all this. I'll disable it for beta3 (that I'll PR today) and give me your opinion there over the generated fixtures.

Reply:
This comment was reffering to `Name: ""` in the `Identifier` -  that should never happen.
False values are a different (but valid) issue.